src/redis-cli.c

```diff
@@ -754,11 +755,22 @@ static int cliReadReply(int output_raw_strings) {
     int output = 1;
 
     if (redisGetReply(context,&_reply) != REDIS_OK) {
+        if (config.monitor_skip) {
+            context->fd = 0;
```

Comment:
```suggestion
            context->fd = REDIS_INVALID_FD;
```
@yoav-steinberg I found the reason. `0` is stdin, so new code will close stdin.
But I can't update the code. Maybe we can open a new PR, and cherry-pick this.

Reply:
https://github.com/huangzhw/redis/commit/c8f040b24fd5ee5057012bff72ca7c21cac4b935

Reply:
It is better. if we still don't get the response from the author. I think you can take it over

Reply:
Yes. Let's wait.

Reply:
i think we can move on (do not get the response)
and open a new PR for this fix